android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/VPNService.kt

```diff
@@ -19,29 +19,29 @@ import com.wireguard.config.Peer
 import com.wireguard.crypto.Key
 import org.json.JSONObject
 import java.util.*
+import org.mozilla.firefox.vpn.daemon.GleanMetrics.Sample
```

Comment:
Yeah, that is a whole different issue. After this whole Glean migration is stable we will audit our data collection choices and the sample category will be gone first, for sure :D

Reply:
> this is also going to start using a new app id and so would be the best opportunity to change it.

Wait, why do you say this will start using a new app id? I did not realize that would be the case.